slither/core/declarations/contract.py

```diff
@@ -663,6 +666,24 @@ def functions_and_modifiers_declared(self) -> List["Function"]:
         """
         return self.functions_declared + self.modifiers_declared  # type: ignore
 
+    @property
```

Comment:
Can we add a test case under `slither/tests/unit/core` to ensure that it works for examples like https://github.com/crytic/slither/issues/1331#issuecomment-1209433713

Reply:
Done!